omod/src/main/webapp/fragments/patient/result.mini.gsp

```diff
@@ -1,2 +1,4 @@
-<img ng-src="${ ui.resourceLink("kenyaui", "images/glyphs/patient_") }{{ patient.gender }}.png" class="ke-glyph" />
-{{ patient.name }} <span style="color: #999">{{ patient.age }}</span>
\ No newline at end of file
+<img ng-src="${ ui.resourceLink("kenyaui", "images/glyphs/patient_") }{{ patient.gender.toLowerCase() }}.png" class="ke-glyph" />
```

Comment:
Why would you need the gender to be in lower case?

Reply:
This gender value is being concatenated with the rest of the file name. The file name is in all lowercase